com.sleepfiles.OSCAR.yaml

```diff
@@ -0,0 +1,69 @@
+######################################################
+# Flatpak manifest example for Open Build Service
+# https://docs.flatpak.org/en/latest/manifests.html
+# Input should be YAML, even though the file can have
+# a .json suffix (JSON is a subset of YAML).
+# Don't use '//' comments!
+######################################################
+
+# Special OBS field because flatpak does not have a version field
+# Default will be '0' if the field is missing.
+#!BuildVersion: 1.3.1
+---
+app-id: com.sleepfiles.OSCAR
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: OSCAR
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
+  - --device=dri
+build-options:
+  ldflags: -L/app/lib
+  cxxflags: -L/app/lib
+  env:
+    - QMAKE_CXXFLAGS: -L/app/lib
+    - QMAKE_LDFLAGS: -L/app/lib
+    
+modules:
+  - name: glu
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://mesa.freedesktop.org/archive/glu/glu-9.0.2.tar.xz
+        sha256: 6e7280ff585c6a1d9dfcdf2fca489251634b3377bfc33c29e4002466a38d02d4
+    cleanup:
+      - /include
+      - /lib/*.a
+      - /lib/*.la
+      - /lib/pkgconfig
+  
+  - name: OSCAR
+    buildsystem: qmake
+    config-opts:
+      - OSCAR_QT.pro
+      - LIBS+=-L/app/lib
+    sources:
+      - type: archive
+        url: https://gitlab.com/pholy/OSCAR-code/-/archive/v1.3.1/OSCAR-code-v1.3.1.tar.bz2
+        sha256: 886aed9d88b33f6429fd271f6e97ab6e8d3d856b4beda67624c1cb5d49460681
+      - type: file
+        path: com.sleepfiles.OSCAR.metainfo.xml
+      - type: file
+        path: com.sleepfiles.OSCAR.desktop
+      - type: file
+        path: com.sleepfiles.OSCAR.png
+    build-commands:
+      # the generated makefiles do not support make install(?)
+      - mkdir -p /app/{bin,share/metainfo,share/icons,share/applications}
+      - install -m 755 oscar/OSCAR /app/bin/OSCAR
+      - install -m 644 Building/Linux/OSCAR.png /app/share/icons/com.sleepfiles.OSCAR.png
```

Comment:
this is the wrong directory for applicaton icons.

Reply:
It should be in the directory `$FLATPAK_DEST/share/icons/hicolor/NxN/apps/` where N is the size of the icon (a 128px or bigger is required)

This is the standard for icons.

Reply:
Fixed.